docker/CISetup.sh

```diff
@@ -1,40 +0,0 @@
-#!/usr/bin/env bash
-# Copyright © 2018 DIG Development team. All rights reserved.
-
-# Hardcoded location of envConfig.json in travis build
-configFile="/home/travis/build/JamesLongman/DIGBot/config/envConfig.json"
-
-# Script arguements, takes bot token, sub-bot tokens and sub-bot IDs
-token=$1
-ID1=$2
-ID2=$3
-ID3=$4
-ID4=$5
-ID5=$6
-ID6=$7
-token1=$8
-token2=$9
-token3=${10}
-token4=${11}
-token5=${12}
-token6=${13}
-
-# Rewrite enviroment variables of envConfig.json pre tests. These lines effectively create
-# and subsequently overrite 13 versions of the config file. There is most likely a much
-# more efficient way of altering the config arguements to cut down on build time
-# Issue #50, build time for these lines: ~26 seconds
-jq -c '.token = $newVal' --arg newVal $token $configFile > tmp.$$.json && mv tmp.$$.json $configFile
```

Comment:
the ."token"/".subBots.subBot1.id" sections here may need to be changed also to match the new config